tests/queries/0_stateless/01470_test_insert_select_asterisk.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS insert_select_dst;
+DROP TABLE IF EXISTS insert_select_src;
+
+CREATE TABLE insert_select_dst (i int, middle_a int, middle_b int, j int) ENGINE = Log;
+
+CREATE TABLE insert_select_src (i int, j int) ENGINE = Log;
+
+INSERT INTO insert_select_src VALUES (1, 2), (3, 4);
+
+INSERT INTO insert_select_dst(* EXCEPT (middle_a, middle_b)) SELECT * FROM insert_select_src;
```

Comment:
We should also add a test for asterisk, compound identifiers, qualified asterisk; or remove support for them.

Reply:
Done.